### PR TITLE
Introduce @magnetis/astro-galaxy-themes

### PR DESCRIPTION
**Pull request by AgtLucas**
# What

This PR introduces a new package: `@magnetis/astro-galaxy-themes`. It also introduces a theme designed for Magnetis' mobile app.

Soon we gonna depreciate the themes inside the core package (`@magnetis/astro-galaxy-core`).

# Why

Another step to bring Astro Galaxy to our mobile app.

### CODE REVIEW

File: packages/themes/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# @magnetis/astro-galaxy-themes
```
Comment (tobiasbu) [inline packages/themes/README.md:1]: 👏

File: packages/themes/src/mobile-app/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { Colors } from '@magnetis/astro-galaxy-tokens';
+
+export type Gradient = [string];
```
Comment (tobiasbu) [inline packages/themes/src/mobile-app/index.d.ts:3]: Oh sorry! I did wrong reading of this type in our conversation. The correct  is `string[]`

Comment (AgtLucas) [reply packages/themes/src/mobile-app/index.d.ts:3]: Ooops ahaha, gonna fix it!

Comment (AgtLucas) [reply packages/themes/src/mobile-app/index.d.ts]: @tobiasbu Done here: https://github.com/magnetis/astro-galaxy/pull/82/commits/2c4852bdd1bef9e0fd913b6e3ad0c74bfa1328b8

**Pull request merged**